lib/routes/routePUT.js

```diff
@@ -104,6 +104,11 @@ export default function routePUT(request, response, log) {
                 }
                 // ETag's hex should always be enclosed in quotes
                 const resMetaHeaders = { ETag: `"${request.calculatedHash}"` };
+                if (data) {
+                    const _data = typeof data === 'object' ?
```

Comment:
Right. It's the very first patch for versioning put to receive a `versionId` back after an `objectPut`. I will need to check with the call path and remove it.